src/components/Header/TopNav/outerItemStates.js

```diff
@@ -31,8 +33,8 @@ const defaultDark = css`
 `
 
 const hoverDark = css`
-  background: #3a3553;
-  color: ${props => props.theme.colors.white};
+  background: ${props => props.theme.colors.vibrant};
+  color: #007f56;
```

Comment:
can't we use a theme var here? :-)

Reply:
Is it worth putting it in the theme, if this is the only place in our codebase where we use this colour?